README.md

```diff
@@ -1,6 +1,6 @@
 ## Welcome
 
-This is a simple markdown file that provides some copy for easy collaborating via Git and GitHub.
+Lets add some new text here. This is a simple markdown file that provides some copy for easy collaborating via Git and GitHub.
```

Comment:
I'm not sure this should say "Lets add some new text". I think we're missing an apostrophe. I'll propose a change.